cypress/integration/centerSpider.spec.js

```diff
@@ -1,17 +1,13 @@
 'use strict';
 
-describe(('Spider application'), () => {
+describe('Spider application', () => {
   before(() => {
-    cy.visit('/');
+    cy.visit('');
```

Comment:
since we have to keep cy.visit, I'd prefer we use '/'. At least it's clear we are visiting the home page. 

cy.visit('') is weird. not clear what URL we visit.

Reply:
roger